client/src/views/devices/GreetDeviceModal.vue

```diff
@@ -169,13 +216,14 @@ const { t } = useI18n();
 const pageStep = ref(GreetDeviceStep.WaitForGuest);
 const canGoForward = ref(false);
 const waitingForGuest = ref(true);
+const isEmailSent = ref(false);
 const greeter = ref(new DeviceGreet());
 // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
 const notificationManager: NotificationManager = inject(NotificationKey)!;
 
 function getTitleAndSubtitle(): [string, string] {
```

Comment:
Not your fault (in fact it was probably mine :p), but it's better to add an interface, something like
```typescript
interface GreetDeviceTitle {
  title: string;
  subtitle?: string;
}

function getTitleAndSubtitle(): GreetDeviceTitle {
  ...
}
```